### DISCUSSION TITLE
Broken Rectangle Pro after updating to 3.0.9 [Ventura 13.5.2]

### DISCUSSION THREAD

**Discussion by JVioleGrace** [rxhanson/RectanglePro-Community#355] [Q&A] [answered]
Hello, I just updated from 3.0.6 to 3.0.9, and then Rectangle Pro window throw stopped working, well it only works on the preferences window of Rectangle Pro now. I have all my settings reset to default.
<img width="872" alt="Screenshot 2023-09-20 at 10 46 30 AM" src="https://github.com/rxhanson/RectanglePro-Community/assets/65801648/a49bf415-7f91-4197-a5b0-b503b4ae62e0">

Please let me know how to fix this.

**Comment by rxhanson** [comment, marked as answer]
This means that the accessibility permissions are now out of sync. To fix this you can try the following:
1. Close Rectangle Pro if it's running
2. In System Settings -> Privacy & Security -> Accessibility disable Rectangle Pro.
3. Still in System Settings, now remove Rectangle Pro with the minus button.
4. Restart your Mac.
5. Launch Rectangle Pro and enable settings for it as prompted.

**Reply by JVioleGrace** [reply]
Thank you so much! Did this then reinstalled and started working. Keep up the good work!
